cargo-dist/src/config.rs

```diff
@@ -503,6 +509,9 @@ impl DistMetadata {
         if tag_namespace.is_some() {
             warn!("package.metadata.dist.tag-namespace is set, but this is only accepted in workspace.metadata (value is being ignored): {}", package_manifest_path);
         }
+        if install_updater.is_some() {
+            warn!("package.metadata.dist.install-updater is set, but this is only accepted in workspace.metadata (value is being ignored): {}", package_manifest_path);
+        }
```

Comment:
*thinks* this seems like it should be an app-scoped config, yeah? since each app gets its own installer-sh (can ship it as top-level, but, it *can* be app-scoped, at least).

Reply:
Oh yes, good call. Changed.